man/fi_msg.3.md

```diff
@@ -288,6 +288,90 @@ fi_sendmsg.
   be used in all multicast transfers, in conjunction with a multicast
   fi_addr_t.
 
+# Buffered Receives
+
+Buffered receives indicate that the networking layer allocates and
+manages the data buffers used to receive network data transfers.  As
+a result, received messages must be copied from the network buffers
+into application buffers for processing.  However, applications can
+avoid this copy if they are able to process the message in place
+(directly from the networking buffers).  Buffered receives are often
```

Comment:
I would suggest removing the reference to utility providers since it is transparent from users and providers other than utility ones can also expose this via the mode bit.

Reply:
reference removed